soc/nuvoton/npcm/common/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024 Nuvoton Technology Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_include_directories(.)
+
+set(NPCM_BIN_NAME ${CONFIG_KERNEL_BIN_NAME}_signed.bin)
```

Comment:
this is not signed, it has a header added and basically copies the same filename as mcuboot-signed images

Reply:
Thanks for reviewing!
Yes, this is not signed, so I changed to .npcm.bin.